src/common/color-element.js

```diff
@@ -137,6 +140,42 @@ const Self = class ColorElement extends NudeElement {
 
 		customElements.define(this.tagName, this);
 	}
+
+	/**
+	 * Resolve a color value and cache it.
+	 * @param {string} value Color value to resolve.
+	 * @param {Element} element Element to get computed style from to resolve the color value.
+	 * @returns {Color | null} Resolved color value or null if the value cannot be resolved.
+	 */
+	static resolveColor (value, element) {
+		try {
+			return Self.Color.get(value);
+		}
+		catch {
```

Comment:
You don't need to nest everything in the `catch`. Since the try will return if it succeeds, you can just have an empty catch here and proceed as normal.